interface/patient_file/summary/stats.php

```diff
@@ -398,11 +398,12 @@ function load_location( location ) {
         $widgetButtonLabel = xl("Edit");
         $oeLink = $GLOBALS['webroot'] . "/controller.php?prescription&list&id=" . attr($pid);
         $widgetButtonLink = 'editScripts("' . $oeLink . '")';
+
         $widgetButtonClass = "iframe rx_modal";
     }
 
     $bodyClass = "summary_item small";
-    $widgetAuth = true;
+    $widgetAuth=acl_check('patients', 'rx', '', 'write');
```

Comment:
hi @sharonco-matrix ,
I think this seems reasonable. I think we should also have the 'addonly' return also have access to the Edit button for rx since this is default setting for the Clinicians role (the Physicians role has access to the 'write').
Thoughts?